releases/v0.6.0.md

```diff
@@ -0,0 +1,65 @@
+# Release Notes for v0.6.0
+Release Date: June 7th, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Support for attesting pod VMs with Azure vTPMs on SEV-SNP
+- Support for using Project Amber as an attestation service
+- Support for Cosign signature validation with s390x
+- Pulling guest images with many layers can no longer cause guest CPU starvation.
+- Attestation Service upgraded to avoid several security issues in Go packages.
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV(-ES)
+- Intel SGX
+
+The following platforms are untested or partially supported:
+- IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+- AMD SEV-SNP
+
+The following platforms are in development:
+- ARM CCA
+
+## Limitations
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * Image signature validation with AMD SEV-ES is not covered by CI.
```

Comment:
Similar I'd be consistent and use dots at the end of all the bullet points or not at all